python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -540,6 +548,7 @@ export default function EditorContainer({
 
   return (
     <AIConfigContext.Provider value={contextValue}>
+      <Notifications />
```

Comment:
What does this do?

Reply:
It's the Mantine component responsible for showing the notifications that are dispatched with `showNotification`. It was just left out accidentally when copying stuff over